tests/unit/moduleapi/cluster.tcl

```diff
@@ -158,6 +159,17 @@ start_cluster 3 0 [list config_lines $modules] {
         assert_error "ERR Can not execute a command 'set' while the cluster is down" {$node1 do_rm_call set x 1}
     }
 
+    test "Verify Module Auth Succeeds when cluster is down" {
+        r acl setuser foo >pwd on ~* &* +@all
+        assert_error "*cluster is down*" {$node1 set x 1}
```

Comment:
We tell users to only rely on the first uppercase word in the error messages, so we can do the same in our test cases. It's used as an error code, while the rest of the text is a descriptive message to humans which can change.

```suggestion
        assert_error "*CLUSTERDOWN*" {$node1 set x 1}
```